### DISCUSSION TITLE
Mapping a combo involving the dpad disables the dpad for the original controller

### DISCUSSION THREAD

**Discussion by zany130** [sezanzeb/input-remapper#890] [Q&A]
I am trying to map  hotkeys on the input -remapper controller on top of the original controller for example: `start + dpad up` to be `vol up` `start + dpad down` be ` vol down` etc

but each of the buttons when pressed by themselves should come from the original controller

this, for some reason, disables the dpad in the original controller. Is this intentional?

no other button exhibits this behavior. For example, I have `start + y` set to `alt+f4`, and both start and y work normally when pressed by themselves. meaning they come from the original non emulated controller

I can create a dpad mapping in the emulated controller and that will work but I want all normal gamepad inputs to come from the original controller

bug? @sezanzeb

**Comment by sezanzeb** [comment]
This has probably been fixed with https://github.com/sezanzeb/input-remapper/pull/1274
